src/CommandingOfficers/AW4/BrennerWolves/BrennerDoR.java

```diff
@@ -24,7 +24,7 @@ public instantiator()
           "Captain of the 12th Battalion. He has an unshakable faith in humanity's goodness, and will aid any in need.\n"));
       infoPages.add(new InfoPage(
           "Base Zone: 3\n" +
-          "Zone Boost: All units +20 DD\n"));
+          "Zone Boost: All units +20 defense\n"));
```

Comment:
Calling him Brenner"DOR" is kinda redundant now. Shrug.

Reply:
Wally**AI** is also redundant. Again, shrug.